go.mod

```diff
@@ -255,5 +255,5 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v0.44.1-celestia
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/libp2p/go-libp2p-pubsub v0.5.7-0.20211029175501-5c90105738cf => github.com/celestiaorg/go-libp2p-pubsub v0.5.7-0.20220325112944-d33a3e5e13d5
```

Comment:
@Wondertan should I also update this dependency?

Reply:
We can tag things on our pubsub fork just to be consistent with tagging. Could you do it?

Reply:
Yes

Reply:
The branch: https://github.com/celestiaorg/go-libp2p-pubsub/tree/old-datastore

Reply:
https://github.com/celestiaorg/go-libp2p-pubsub/releases/tag/v0.5.6-celestia